config/deployer.default.json

```diff
@@ -0,0 +1,21 @@
+{
+    "ClusterName": "loadtest",
+    "AppInstanceCount": 1,
+    "SSHPublicKey": "~/.ssh/id_rsa.pub",
+    "DBInstanceCount": 1,
+    "DBInstanceEngine": "mysql",
+    "DBInstanceClass": "db.r4.large",
+    "DBUserName": "mmuser",
+    "DBPassword": "mostest80098bigpass_",
+    "MattermostDownloadURL": "https://releases.mattermost.com/5.20.1/mattermost-5.20.1-linux-amd64.tar.gz",
+    "MattermostLicenseFile": "",
+    "LogSettings": {
+        "EnableConsole": true,
+        "ConsoleLevel": "INFO",
+        "ConsoleJson": true,
```

Comment:
If we set to output JSON as default maybe we can run terraform with `-no-color`.  
Error codes make the output really hard to read.

Reply:
Oh .. I always pipe it through logrus. Never was an issue for me. :P 

This is mainly a tool that the user will run and look at the output. I think colors will help. We can disable JSON by default if you prefer. Or users always have the choice to pipe it through logrus. Maybe we can document that more prominently.

Reply:
Fine with disabling JSON for this.

Reply:
done